test/guardian_db_test.exs

```diff
@@ -16,12 +16,12 @@ defmodule GuardianDbTest do
   end
 
   test "after_encode_and_sign_in is successful", context do
-    token = Repo.get(Token, "token-uuid")
+    token = Token.query_schema() |> Repo.get("token-uuid")
```

Comment:
I know could be call without pipe but this is cleaner in my opinion

Reply:
Given how often this is used, this would probably be fine as a function don't you think?

```elixir
def test_token(field // "token-uuid"), do: Repo.get(Token.query_schema(), key)
```

Reply:
@doomspork nobody got time for that (joke) 

let me add it